doc/source/reference/c-api.ufunc.rst

```diff
@@ -124,9 +122,7 @@ Functions
     :param unused:
         Unused and present for backwards compatibility of the C-API.
 
-.. c:function:: PyObject* PyUFunc_FromFuncAndDataAndSignature(PyUFuncGenericFunction* func,
-   void** data, char* types, int ntypes, int nin, int nout, int identity,
-   char* name, char* doc, int unused, char *signature)
```

Comment:
I haven't checked but [`c-api.iterator`](https://github.com/numpy/numpy/blame/master/doc/source/reference/c-api.iterator.rst#L285) also contains longish lines. I can see what happens with backslashes. Is there a line-length-limit for `.rst` files?